mcsema/BC/Lift.cpp

```diff
@@ -219,7 +219,11 @@ static InstTransResult LiftInstIntoBlockImpl(TranslationContext &ctx,
     std::cerr << "Error translating instruction at " << std::hex
               << ctx.natI->get_loc() << "; unsupported opcode " << std::dec
               << opcode << std::endl;
-    return TranslateErrorUnsupported;
+
```

Comment:
Yes. It's kind of like: "I consent to to this approach, knowing that it may not work in all cases."